src/display/editor/tools.js

```diff
@@ -1922,6 +1922,8 @@ class AnnotationEditorUIManager {
   /**
    * Change the editor mode (None, FreeText, Ink, ...)
    * @param {number} mode
+   * @param {boolean} [isFromUser] - true if the mode change is due to a
```

Comment:
Required arguments should be listed before optional ones, so please swap `isFromUser` with `editId` here and in the function definition below.